spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature 'Home page' do 
+  context 'authorized user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.